regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -46,6 +47,16 @@ var PreambleView = ChildView.extend({
     );
   },
 
+  openCitation: function(e) {
+    var $target = $(e.currentTarget);
+    var hash = $target.attr('href');
```

Comment:
What's generating the link markup? Because if it's _supposed_ to link like `href="/preamble/0000_0000/III##0000_0000-III-B"` in the HTML, then is it the case that the markup is updated (then we don't need JS), or that the JS should update the hash to that href?